dlinfer/graph/dicp/vendor/AtbGraph/codegen/atb_graph.py

```diff
@@ -281,27 +281,36 @@ def get_input_data_node(node_list, node_name):
 def make_output_tensor_desc(
     output_names,
     output_data_nodes,
+    optimized_graph,
 ):
     output_tensor_descs = {"param": {}, "create": {}}
 
-    def process_node(output_name, node, input_name=None, need_reshape=False):
+    def process_node(output_name, node, atb_op, input_name=None, need_reshape=False):
         dims, dim_num = get_shape(node)
         dims_str = f'[{",".join(dims)}]'
         dtype = get_dtype(node)
-        info = f""" {{"format": {AclFormat.ACL_FORMAT_ND.value}, "dtype": {dtype}, "dimNum": {dim_num}, "dims": {dims_str} }} """
+        # TransdataOperation transdataType is 2 means convert from ND to FractalNZ
+        out_format = (
+            AclFormat.ACL_FORMAT_FRACTAL_NZ.value
```

Comment:
Here it's better not to judge whether it's a TransdataOperation, but to add an output format attribute to each Operation, which can default to ND and be set to NZ if needed. In atb_op.py, set the output of the TransdataOperation operator to NZ.